src/lib/auth/index.ts

```diff
@@ -19,7 +21,7 @@ export const {
     newUser: "/dashboard",
     error: "/error",
   },
-  adapter: PrismaAdapter(db),
+  adapter: DrizzleAdapter(db) as Adapter,
```

Comment:
This may help:

```typescript
adapter: DrizzleAdapter(db, {
    usersTable: user,
    accountsTable: account,
    sessionsTable: session,
  }),
```

Reply:
I've implemented this but am getting this:

Type 'PgTableWithColumns<{ name: "user"; schema: undefined; columns: { id: PgColumn<{ name: "id"; tableName: "user"; dataType: "string"; columnType: "PgUUID"; data: string; driverParam: string; notNull: true; hasDefault: true; enumValues: undefined; baseColumn: never; }, {}, {}>; ... 6 more ...; updatedAt: PgColumn<...>; ...' is not assignable to type 'DefaultPostgresUsersTable'.
  Type 'PgTableWithColumns<{ name: "user"; schema: undefined; columns: { id: PgColumn<{ name: "id"; tableName: "user"; dataType: "string"; columnType: "PgUUID"; data: string; driverParam: string; notNull: true; hasDefault: true; enumValues: undefined; baseColumn: never; }, {}, {}>; ... 6 more ...; updatedAt: PgColumn<...>; ...' is not assignable to type 'PgTable<{ name: string; columns: { id: DefaultPostgresColumn<{ columnType: "PgText" | "PgUUID" | "PgVarchar"; isPrimaryKey: true; data: string; notNull: true; dataType: "string"; }>; name: DefaultPostgresColumn<...>; email: DefaultPostgresColumn<...>; emailVerified: DefaultPostgresColumn<...>; image: DefaultPostgres...'.
    The types of '_.config.columns.id' are incompatible between these types.